activities/procurement/index.md

```diff
@@ -9,3 +9,7 @@ To fulfill our mission, we need to get things and services from others.
 How we procure:
 
 * [Procurement principles](principles.md)
+
+Procurement process:
+
+* [Procurement process](process.md)
```

Comment:
I think these two can be in the same list, I'd argue 'procurement process' still falls in 'how we procure'. If you don't think so feel free to come up with a better name for this list.

```suggestion
* [Procurement process](process.md)
```